.github/workflows/docker.yml

```diff
@@ -4,7 +4,7 @@ on:
   schedule:
     - cron: "0 10 * * *" # everyday at 10am
   push:
-    branches: ["master", "release-*"]
+    branches: ["master", "release-*", "torch_113_image_upgrade"]
```

Comment:
Will want to remove this before landing, of course (just calling it out for completeness ;) ).

Reply:
Haha absolutely! Will do one final round of testing with all of the updates and remove this :)